zfmt/ast.go

```diff
@@ -443,6 +442,21 @@ func (c *canon) op(p ast.Op) {
 		c.ret()
 		c.flush()
 		c.write(")")
+	case *ast.Pool:
+		c.next()
+		c.open("")
+		c.pool(p, true)
```

Comment:
`true` is pretty opaque here. How about something more explicit?
```suggestion
		c.write("from ")
		c.pool(p)
```